### DISCUSSION TITLE
Warfare Unit Sheets? Relationships Tab?

### DISCUSSION THREAD

**Discussion by TitanicViking** [3darkman/foundryvtt-chroniclesystem#69] [Ideas]
Good to see the project active again! Back when I was running my ASOIF game, one of our players did some tinkering with the system & added in some warfare sheets & also added the relationships tab I had suggested a while back. I've still got the files, so I could send them over to you if you wanted to take a look at them? 

The relationships tab was more or less a drag & drop with an area for notes to drop other character sheets onto. Helped a ton with tracking long term dispositions & effects of intrigues.

**Comment by 3darkman** [comment]
Hi, the unit sheets are almost ready, I started doing them a long time ago, but due to lack of time, I put them aside. But of course, you can send them to me, who knows, maybe they’ll give me some more ideas.

**Reply by TitanicViking** [reply]
Awesome. Where would a good place be to send the changes? Do you have discord or your email? I didnt do them myself, but one of my players was a programmer & did a fairly good job of making changes.

**Reply by 3darkman** [reply]
I'm there on the green ronin discord, in the project's system.json, there's my discord too.
